.travis.yml

```diff
@@ -1,2 +1,10 @@
 language: ruby
+rvm:
+- 2.1
 script: "bundle exec jekyll build"
```

Comment:
Also are we absolutely sure that jekyll is deployed on the travis infrastructure?  It seems to make sense to have an

install:
 - bundle install